openmrs/i18n/registration/locale_fr.json

```diff
@@ -21,5 +21,7 @@
   "REGISTRATION_INSTITUTE_ADDRESS": "Hopital de la Cité Verte, District - Yaoundé",
   "FEE_INFORMATION_LOCALE_KEY": "Frais d'Information",
   "NUTRITIONAL_VALUES_LOCALE_KEY": "Valeurs Nutritionnelles",
-  "REGISTRATION_PRINT_WITH_BARCODE": "Impression avec <u>C</u>ode Barre"
+  "REGISTRATION_PRINT_WITH_BARCODE": "Impression avec <u>C</u>ode Barre",
+  "REGISTRATION_SEARCH_ADDRESS_LABEL_KEY": "Quartier",
+  "REGISTRATION_SEARCH_ADDRESS_PLACEHOLDER_KEY": "Saisissez le nom du quartier"
```

Comment:
@rubailly can we use the 'camel case' here as well for consistency? i.e. `Saisissez le Nom du Quartier`